### ISSUE TITLE
Windows - GP starts up but doesnt appear

### ISSUE THREAD

**Issue by MRMRJ** [bardsoftware/ganttproject#1230]
I know this sounds daft, and it might well be my own configuration that's causing it, but... So the issue I've got is that GanntProject starts up it appears in the taskbar but the GanttProject workspace/window does not appear. It's almost as if it's been left on another extended screen in a dual-screen arrangement but I've tried extending and distending to no avail. Ive searched the various places for an answer to no avail so I thought I should take a moment to report it here - sorry if I've missed something really obvious.
Windows 7sp1 64bit
MSI 6837d laptop

**Comment by dbarashev** [comment]
Please share the contents of `ganttproject.log` file after starting GanttProject. The log file sits in [your home directory](https://en.wikipedia.org/wiki/Home_directory#Default_home_directory_per_operating_system)

**Comment by MRMRJ** [comment]
Thanks for your reply - here's the file - I hope it helps
[ganttproject.txt](https://github.com/bardsoftware/ganttproject/files/208339/ganttproject.txt)

**Comment by dbarashev** [comment]
The log says that something caused GanttProject window to close. Was it your action in the task bar? 

Anyway, can you also send `.ganttproject` file from your home dir? It contains settings, including GanttProject window position on the screen and its size. 

Notice the dot in the file name. It might be hidden file in Windows

**Comment by MRMRJ** [comment]
Yes - you are right - the only way to shut the program down was to terminate it from the taskbar (or task manager but taskbar was quicker)

[- Copy.ganttproject.txt](https://github.com/bardsoftware/ganttproject/files/208368/-.Copy.ganttproject.txt)

**Comment by dbarashev** [comment]
So it sits off the screen indeed:

    <geometry x="-1132" y="129" width="962" height="743"/>

You can edit the settings file yourself in any text editor or use the attached one where I replaced -1132 with "0"

[-.Copy.ganttproject.txt](https://github.com/bardsoftware/ganttproject/files/208387/-.Copy.ganttproject.txt)

**Comment by MRMRJ** [comment]
Ta-daaah! Awesome! Thanks! I have no idea how it got over there, or, more properly, why it wouldnt show up there when I extended my desktop... unless the 2nd time I tried it the screen extended to the right instead of where it was before... which is possible given the various ways I hook up my laptop.... Anyway, I'll let you work out how this might be avoided in future but at least I know where to look and what to do to fix it. Thanks again!
